test/regression/api/canary/content/posts_spec.js

```diff
@@ -264,11 +264,6 @@ describe('api/canary/content/posts', function () {
         let paidPost;
         let membersPostWithPaywallCard;
 
-        before(function () {
-            // NOTE: ideally this would be set through Admin API request not a stub
-            sinon.stub(labs, 'isSet').withArgs('members').returns(true);
-        });
-
```

Comment:
Removed this as it's no longer needed (`members` is always `true`) and was interfering with `labs.isSet()` for other features.